SPECS-EXTENDED/xdp-tools/xdp-tools.signatures.json

```diff
@@ -0,0 +1,5 @@
+{
+ "Signatures": {
+  "xdp-tools-1.2.9.tar.gz": "3f8d30bd2e00c522103d224014f59a95400144aba1f3b322c6ad473541a0f99e",
```

Comment:
Looks like the SRPM packer doesn't like the trailing comma here. Logs are below:

```
ERRO[0006] Failed to read the signatures file (/specs/xdp-tools/xdp-tools.signatures.json): invalid character '}' looking for beginning of object key string. 
ERRO[0006] Failed to pack (/specs/xdp-tools/xdp-tools.spec). Error: invalid character '}' looking for beginning of object key string 
PANI[0006] invalid character '}' looking for beginning of object key string 

```